src/Label305/DocxExtractor/Decorated/Extractors/RNodeExtractor.php

```diff
@@ -74,16 +74,8 @@ private function parseChildNode(
 
         } elseif ($rChild instanceof DOMElement && in_array($rChild->nodeName, ["w:r", "w:smartTag"])) {
             foreach ($rChild->childNodes as $propertyNode) {
-                if ($propertyNode instanceof DOMElement && $propertyNode->nodeName == "w:t") {
-                    if ($propertyNode->getAttribute("xml:space") == 'preserve') {
-                        $text = implode($this->parseText($propertyNode));
-                    } else {
-                        $text = trim(implode($this->parseText($propertyNode)), " ");
-                    }
-                } else {
-                    $this->parseChildNode($propertyNode, $result, $webHidden, $bold, $italic, $underline, $brCount, $highLight,
-                        $superscript, $subscript, $text, $style);
-                }
+                $this->parseChildNode($propertyNode, $result, $webHidden, $bold, $italic, $underline, $brCount, $highLight,
```

Comment:
Because "w:t" is also processed in the parseChildNode function there is no need to copy it here